e2e/cypress/integration/dashboard-xhr.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { interceptCompanionUrlRequest, interceptCompanionUnsplashRequest, runRemoteUrlImageUploadTest, runRemoteUnsplashUploadTest } from './util.mjs'
```

Comment:
I have a small allergy for folders and files named `util` 😅 

Shall we rename this to `reused-tests.ts` (note also the change from `mjs` to `ts`)?